wbooks/src/app/scss/app.scss

```diff
@@ -0,0 +1,66 @@
+.sign-up-container {
+  background-color: $wild-sand;
+  border-top: 8px solid $cerulean;
+  display: flex;
+  flex-flow: column wrap;
+  height: 100%;
+  justify-content: center;
+  max-height: 700px;
+  max-width: 300px;
+  padding: 20px 24px 24px;
+  width: 100%;
+
+  .wolox-logo {
+    height: 70px;
+    max-height: 100%;
+    max-width: 212px;
+    width: 100%;
+  }
+
+  .sign-up-form {
+    border-bottom: 2px solid $grey;
+    display: flex;
+    flex-flow: column wrap;
+    flex-grow: 1;
+    margin-top: 16px;
+    padding-bottom: 16px;
+    width: 100%;
+
+    .sign-up-label {
+      font-size: $font-small;
+      margin-bottom: 8px;
+    } 
+
+    .input-form {
+      background-color: $white;
+      border-radius: 5px;
+      font-size: $font-small;
+      height: 100%;
+      margin-bottom: 16px;
+      max-height: 32px;
+      max-width: 252px;
+      padding: 8px 0;
+      width: 100%;
+    }
+
+    .sign-up-button {
+      background-color: $sulu-green;
+      border-bottom: 2px solid $grey;
+      border-radius: 5px;
+      color: $wild-sand;
+      font-size: $font-small;
+      margin-top: 8px;
```

Comment:
Remember that if you can use margin-bottom,  don't use margin-top

Reply:
Changed